package.json

```diff
@@ -67,6 +67,7 @@
     "react-native": "0.63.3",
     "react-native-testing-library": "^6.0.0",
     "react-test-renderer": "~17.0.1",
+    "rollup-plugin-svg": "^2.0.0",
```

Comment:
Why? We are already using a svg plugin

Reply:
Isn’t `@rollup/plugin-url` enough?

Reply:
I tried to use that one, but couldn't make it work. I did find an [issue](https://github.com/rollup/plugins/pull/158) about it where they decided that they will not implement what we need

Reply:
If the plug-in provides you the uri (base64) you can pass that to the image property in the structure mode using base64 attribute. Just check the typings :)

Reply:
It isn't base64, but I managed to make it work anyways :)